### ISSUE TITLE
[python, 3.5.2/post1] Pip source distribution does not contain cc files

### ISSUE THREAD

**Issue by mcneish1** [google/protobuf#4605]
Possibly related: #4383

```console
$ wget https://pypi.io/packages/source/p/protobuf/protobuf-3.5.2.post1.tar.gz
$ tar tf protobuf-3.5.2.post1.tar.gz | grep cc
```
and 

```console
$ wget https://pypi.io/packages/source/p/protobuf/protobuf-3.5.2.tar.gz
$ tar tf protobuf-3.5.2.tar.gz | grep cc
```
However
```console
$ wget https://pypi.io/packages/source/p/protobuf/protobuf-3.5.1.tar.gz
$ tar tf protobuf-3.5.1.tar.gz | grep cc
protobuf-3.5.1/google/protobuf/pyext/extension_dict.cc
protobuf-3.5.1/google/protobuf/pyext/descriptor_database.cc
protobuf-3.5.1/google/protobuf/pyext/descriptor_pool.cc
protobuf-3.5.1/google/protobuf/pyext/descriptor.cc
protobuf-3.5.1/google/protobuf/pyext/repeated_composite_container.cc
protobuf-3.5.1/google/protobuf/pyext/repeated_scalar_container.cc
protobuf-3.5.1/google/protobuf/pyext/descriptor_containers.cc
protobuf-3.5.1/google/protobuf/pyext/message_module.cc
protobuf-3.5.1/google/protobuf/pyext/message.cc
protobuf-3.5.1/google/protobuf/pyext/map_container.cc
protobuf-3.5.1/google/protobuf/pyext/message_factory.cc
protobuf-3.5.1/google/protobuf/internal/api_implementation.cc
protobuf-3.5.1/google/protobuf/internal/python_protobuf.cc
```

Is there some reason these files weren't packaged for 3.5.2?

**Comment by TeBoring** [comment]
We have already provided wheel on pypi. Could you use that?

**Comment by mcneish1** [comment]
No, the machine I'm trying to use this on is a ppc64le machine.

It's not really an issue, per se, as I can grab the whole protobuf repository instead. It just seemed like an omission.

**Comment by TeBoring** [comment]
fixed in 3.6.0. https://pypi.org/project/protobuf/#files
